x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -132,14 +157,14 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
     }, [entry, onVisited]);
 
     return (
-      <EuiFlexGroup
+      <StyledEuiFlexGroup
         gutterSize="s"
         alignItems="center"
         direction="row"
```

Comment:
For example (re: my prior comment above): does this prop `direction` do anything?